ms/second-review-responses.tex

```diff
@@ -166,23 +196,65 @@
 I am fairly pleased overall with the revised version of this article. I commend the authors for the substantial amount of work conducted. This contribution now convincingly demonstrates how important it is to properly account for population structure when individuals are distributed along a spatial continuum.
 \end{quote}
 
+Thanks. We are also fairly pleased with it, overall.
+\plr{this is probably too much =)}
+
 \begin{point}{}
 A couple of minor points about the forward model are still puzzling me though. First of all, in one dimension, the variance of $L(o)$, the random variable corresponding to the offspring's spatial position, given $l(p)$, the parent's position, is equal to $E((L(o)-l(p))^2):=\sigma^2$. Hence, $\sigma$ is here the square root of the expected squared Euclidean distance between parents and offspring. In general, assuming isotropic migrations, $\sigma$ is the square root of the expected squared Euclidean distance taken along any dimension. It seems to me that this definition is distinct from the mean parent-offspring distance, as stated by the authors.
 \end{point}
 
 \reply{
-Good point here. We have corrected this to ``$\sigma$ is the mean parent--offspring distance along each of the x and y axes'' and now direct readers to \citep{Rousset1997} for discussion of how these parameters are defined in one- and two-dimensional habitats. The mean parent-offspring distance in two dimensions is $\sigma\sqrt{2}$, but we follow previous work in defining neighborhood size as $4 \pi \rho \sigma^2$ (i.e. without the factor of $\sqrt{2}$; see \citep{Rousset1997} for details). \llname{axialdist}
+Good point here.
+We did intend to define $\sigma$ as is usual in the literature,
+so have corrected the wording
+and now direct readers to \citep{Rousset1997} for discussion of how these parameters are defined in one- and two-dimensional habitats. \llname{axialdist}
```

Comment:
I think it's clear R3 understands things correctly, so we don't need to explain things in the response.